index.js

```diff
@@ -22,6 +26,11 @@ module.exports = postcss.plugin("postcss-calc", function(options) {
       helpers.try(function transformCSSCalc() {
         var value = reduceCSSCalc(decl.value, precision)
 
+        if (throwWhenCannotReduce && CONTAINS_CALC.test(value)) {
+          throw decl.error("Could not reduce expression: " + decl.value,
```

Comment:
Here is an example on how to test messages https://github.com/postcss/postcss-custom-properties/blob/master/test/index.js#L74-L80